.github/PULL_REQUEST_TEMPLATE/small-change.md

```diff
@@ -9,6 +9,12 @@
 
 Not applicable.
 
+## Effects on driving behavior
+
+<!-- Describe how this PR affects the outputs of driving behavior. -->
```

Comment:
Related to https://github.com/autowarefoundation/autoware/pull/3423/files#r1164888354.

```suggestion
<!-- Describe how this PR affects the system behavior. -->
```